src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -1129,6 +1131,7 @@ trait Types
     override def safeToString: String = "<error>"
     override def narrow: Type = this
     override def kind = "ErrorType"
+    override def mapOver(map: TypeMap): Type = this
```

Comment:
There are many `override def mapOver(map: TypeMap): Type = this` added in this commit, and they are unnecessary in principle. I suggest to either make `mapOver` in `Type` abstract or throw, or remove all redundant overrides.

Reply:
Hmm, I can't remember the sequence of how I arrived at this. I do remember running into problems when `Type` became a SAM and the inliner (prior to #5895) was overzealous. Maybe I made this non-abstract as a workaround.